traffic_portal/test/end_to_end/deliveryServices/delivery-services-spec.js

```diff
@@ -327,6 +331,30 @@ describe('Traffic Portal Delivery Services Suite', function() {
 		});
 	});
 
+	it('should add a required Static DNS entry to the HTTP delivery service', function() {
+		pageData.dsLink.click();
+		console.log('Adding Static DNS entry to ' + mockVals.httpXmlId);
+		pageData.moreBtn.click();
+		pageData.viewStaticCapabilitiesMenuItem.click();
+		expect(browser.getCurrentUrl().then(commonFunctions.urlPath)).toMatch(commonFunctions.urlPath(browser.baseUrl)+"#!/delivery-services/[0-9]+/static-dns-entries");
+		pageData.addStaticDNSBtn.click();
+		// expect(pageData.selectFormSubmitButton.isEnabled()).toBe(false);
+		// set host name
+		pageData.host.sendKeys(mockVals.staticDNShostName);
+		// set type ID
```

Comment:
want to say in your comment: `set type to CNAME_RECORD's id` or something so it's clear what `3` is.

Reply:
Change completed.